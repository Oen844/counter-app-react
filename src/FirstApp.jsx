export function FirsApp() {
  return (
    <h1>First App</h1>
  )
}