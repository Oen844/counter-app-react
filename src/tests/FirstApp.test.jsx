import { render } from "@testing-library/react"
import { FirstApp } from "../FirstApp"



describe('Pruebas en <FristApp/>', () => {

    test('debe de hacer match con el snapshot', () => {

        const title = 'Hola, soy Goku!';

        const {container} = render( <FirstApp title= {title} /> );

        expect( container ).toMatchSnapshot();
    })

    test('Debe de mostrar en el titulo en una h1', () => {
        const title = 'Hola, soy Goku!';

        const { container, getByText } = render( <FirstApp title= {title} /> );

        expect ( getByText(title) ).toBeTruthy();

        const h1 = container.querySelector('h1');
        console.log('**************************************')
        console.log(h1.innerHTML);

        expect(h1.innerHTML).toContain( title );

    })

})