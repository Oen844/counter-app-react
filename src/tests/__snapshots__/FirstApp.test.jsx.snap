// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <FristApp/> debe de hacer match con el snapshot 1`] = `
<div>
  <h1>
    Hola, soy Goku!
  </h1>
  <p>
    Soy un guerrero
  </p>
  <p>
    Pau!!!!
  </p>
</div>
`;
